shotover-proxy/tests/cassandra_int_tests/routing.rs

```diff
@@ -204,7 +229,13 @@ mod composite_key {
 
     async fn simple_test(shotover: &CassandraConnection, cassandra: &CassandraConnection) {
         create_keyspace(shotover).await;
+        if shotover.is(&[CassandraDriver::Scylla]) {
+            tokio::time::sleep(std::time::Duration::from_secs(20)).await;
+        }
         create_table(shotover).await;
+        if shotover.is(&[CassandraDriver::Scylla]) {
```

Comment:
Are we better off not introducing a workaround just to update? Might as well just wait for the fix

Reply:
Oh, I agree, I just wanted to confirm that was the problem.

Also handy to have around in case we do need to land this early for some other scylla fix we need.